README.md

```diff
@@ -173,6 +173,21 @@ By default, py_zipkin uses a thread local storage for the attributes, which is
 defined in `py_zipkin.stack.ThreadLocalStack`.
 
 
+Firehose mode [EXPERIMENTAL]
+----------------------------
+
+"Firehose mode" records 100% of the spans, regardless of
+sampling rate. This is useful if you want to treat these spans
+differently, e.g. send them to a different backend that has limited
+retention. It works in tandem with normal operation, however there may
+be additional overhead. In order to use this, you add a
+`firehose_handler` just like you add a `transport_handler`.
+
```

Comment:
If someone wanted to enable only firehose mode and disable the normal transport, they could simply set the sampling rate to 0% and provide the `firehose_handler`, correct? Could be worth mentioning that here.

Reply:
I feel like this is implied in "regardless of sampling rate". I'm gonna merge as-is. We'll need to add additional docs about firehose anyway (e.g. example handler and proxy daemon) so if this still isn't clear I'll fix it up later